lib/css/components/_stacks-prose.less

```diff
@@ -66,34 +66,34 @@
 
     h1,
     h1 code {
-        font-size: @fs-headline2;
+        font-size: @fs-headline1;
         margin-bottom: .2em;
     }
 
     h2,
     h2 code {
-        font-size: @fs-headline1;
+        font-size: @fs-title;
         margin-bottom: .3em;
     }
 
     h3,
     h3 code {
-        font-size: @fs-title;
+        font-size: @fs-subheading;
     }
 
     h4,
     h4 code {
-        font-size: @fs-subheading;
+        font-size: @fs-body3;
     }
     
     h5,
     h5 code {
-        font-size: @fs-body3;
+        font-size: @fs-body2;
     }
     
     h6,
     h6 code {
-        font-size: @fs-body2;
+        font-size: @fs-body1;
```

Comment:
The thing that annoys me about this is that this header is now smaller than our body text 🤔

Reply:
Agreed. Should we set to the same size as `h5` or just not worry about it? `h6` isn't really supported on prod anyways, so I'm not *that* worried about the styles for it.

Reply:
Something to revisit, for sure. Not holding up the show. Just want to make the point that this could consume any markdown source, and not _just_ markdown from our editor, ya know?